targets/simple_switch/simple_switch.h

```diff
@@ -95,14 +95,14 @@ class SimpleSwitch : public Switch {
     return mirroring_map.erase(mirror_id);
   }
 
-  int mirroring_mapping_get(mirror_id_t mirror_id) const {
-    return get_mirroring_mapping(mirror_id);
+  bool mirroring_mapping_get(mirror_id_t mirror_id, port_t *port) {
```

Comment:
by the way it doesn't seem right to be dropping the `const` here?

Reply:
Fixed